src/asm/math.c

```diff
@@ -122,6 +122,22 @@ int32_t math_Div(int32_t i, int32_t j)
 	return double2fx(fx2double(i) / fx2double(j));
 }
 
+/*
+ * Power
+ */
+int32_t math_Pow(int32_t i, int32_t j)
+{
+	return double2fx(pow(fx2double(i), fx2double(j)));
```

Comment:
Should we, instead of converting to floating-point and possibly suffering precision issues, perform the computations in fixed-point directly? (If that's even possible)

Reply:
Maybe as a separate issue+PR, but all the other functions are wrappers around floating-point math.h ones.